modules/tinymce/src/plugins/table/main/ts/selection/Ephemera.ts

```diff
@@ -7,21 +7,19 @@
 
 import { Fun } from '@ephox/katamari';
 
-const selected = 'data-mce-selected';
-const selectedSelector = 'td[' + selected + '],th[' + selected + ']';
+const sSelected = 'data-mce-selected';
```

Comment:
We probably should just remove all the thunks here, but that's a task for another day.

Reply:
One side note though, is maybe we should name these variables differently? As normally variables that start with `s` are agar steps.

Reply:
Yeah, I just reached for Hungarian Notation as a quick fix. Probably the best option is unthunking them.

Reply:
I could use `str` as a prefix...?

Reply:
Sounds good to me :+1:

Reply:
Done.

Reply:
These thunks are all from the Bolt era. If we un-thunk them the strings may even be inlined?